docs/pages/tutorials/minimal/extend_schema.mdx

```diff
@@ -0,0 +1,412 @@
+So if you followed the directions, you need to run this command in the root directory.
+
+```sh
+pnpm mud set-version --tag main && pnpm install
+```
+
+## Modify the MUD configuration file
+
+1. In an editor, open `packages/contracts/mud.config.ts` and add this table definition:
+
+   ```ts
+      History: {
+        keySchema: {
+          counterValue: "uint32",
+        },
+        schema: {
+          blockNumber: "uint256",
+          time: "uint256"
+        }
+      },
+   ```
+
+<details>
+<summary>The complete file</summary>
+
+```ts
+import { mudConfig } from "@latticexyz/world/register";
+
+export default mudConfig({
+  tables: {
+    Counter: {
+      keySchema: {},
+      schema: "uint32",
+    },
+    History: {
+      keySchema: {
+        counterValue: "uint32",
+      },
+      schema: {
+        blockNumber: "uint256",
+        time: "uint256",
+      },
+    },
+  },
+});
+```
+
+</details>
+
+<details>
+<summary>Explanation</summary>
+
+A MUD table has two schemas:
+
+- `keySchema`, the key used to find entries
+- `schema`, the value in the entry
+
+Each schema is represented is a structure with field names as keys, and the appropriate [Solidity data types](https://docs.soliditylang.org/en/latest/types.html) as their values.
+
+In this case, the counter value is represented as a 32 bit unsigned integer, because that is what `Counter` uses.
+The block number and time are both the results of opcodes, so they are represented by the standard Solidity data type, `uint256` (unsigned 256 bit integer).
+
+</details>
+
+2. Run this command in `packages/contracts` to regenerate the code.
+
+   ```sh
+   pnpm build:mud
+   ```
+
+## Update `IncrementSystem`
+
+1. In an editor, open `packages/contracts/src/systems/IncrementSystem.sol`.
+
+   - Modify the second `import` line to import `History`.
+
+     ```solidity
+     import { Counter, History, HistoryData } from "../codegen/Tables.sol";
+     ```
+
+   - Modify the `increment` function to also update `History` by adding this line just before `return newValue`.
+
+     ```solidity
+     History.set(newValue, block.number, block.timestamp);
+     ```
+
+     To see the exact functions that are available, you can look at `packages/contracts/src/codegen/tables/History.sol` (that is the reason we ran `pnpm build:mud` to recreate it already).
+
+<details>
+   <summary>
+     The complete file
+   </summary>
+
+     ```solidity
+     // SPDX-License-Identifier: MIT
+     pragma solidity >=0.8.0;
+
+     import { System } from "@latticexyz/world/src/System.sol";
+     import { Counter, History, HistoryData } from "../codegen/Tables.sol";
+
+     contract IncrementSystem is System {
+       function increment() public returns (uint32) {
+         uint32 counter = Counter.get();
+         uint32 newValue = counter + 1;
+         Counter.set(newValue);
+         History.set(newValue, block.number, block.timestamp);
+         return newValue;
+       }
+     }
+     ```
+
+</details>
```

Comment:
The Emojimon tutorial has some examples of how you can use a bigger, collapsed code block to "focus" on the changed lines but still allow for expanding+copying the entire file. We should use that here!

https://mud.dev/tutorials/emojimon/players-and-movement#11-create-the-components-as-tables

Reply:
Thank you, fixed.